docs/build_docs.py

```diff
@@ -103,10 +98,35 @@ def parse_args_with_argparse():
         default=None,
         help="Output dir to write built docs to",
     )
+    parser.add_argument(
+        "--repo-path",
+        type=str,
+        default=None,
+        help="Path to the repository (e.g., /path/to/geoips)."
+        "If not provided uses [package_name] to find the packages directory"
+        "Must be an existing directory.",
+    )
 
     # Parse arguments
     args = parser.parse_args()
 
+    if not args.repo_path:
+        try:
+            package_path = os.path.dirname(
+                str(importlib.resources.files(args.package_name))
```

Comment:
Getting this error otherwise.
![Screenshot 2024-11-14 at 2 48 27 PM](https://github.com/user-attachments/assets/0626f4ca-ae62-439c-bbc7-49a533511210)

Reply:
Oh weird! I wasn't getting that on my end. I'll change it :)

Reply:
Changed! :)